react/features/filmstrip/constants.js

```diff
@@ -150,14 +145,14 @@ export const TILE_VERTICAL_CONTAINER_HORIZONTAL_MARGIN = 2;
  *
  * @type {number}
  */
-export const TILE_VIEW_GRID_VERTICAL_MARGIN = 12;
+export const TILE_VIEW_GRID_VERTICAL_MARGIN = 14;
 
 /**
  * The horizontal margin of the tile grid container.
  *
  * @type {number}
  */
-export const TILE_VIEW_GRID_HORIZONTAL_MARGIN = 12;
+export const TILE_VIEW_GRID_HORIZONTAL_MARGIN = 14;
```

Comment:
Was this increased so that we would have consistent margins when the scrollbar is visible? Because I saw on the new designs that we actually want to reduce the margins

Reply:
Yes exactly. @yanas approved the change.

I guess when we are implementing the new design we should discuss what to do with the scroll. Anyway I think there are lot of issues currently with the scrolls:
 - The color of the scroll matches the background of the page and it is not visible in tile view.
 - If we don't match the spacing to the scroll size when a scroll appears the tiles are going to be moved to the left and the spacing will look weird because the right side will be always bigger than the left one.
 - We are not controlling the scroll size on FF (not sure if it is possible). And the scroll on FF behaves differently.

In addition the complexity of the algorithm increases so much if we need to account for the scroll size.

I think we need to think about the scroll and work on it separately in order to fix those issues. Some options that I have in mind are to not show the scroll at all or maybe to decrease it size for the new designs (although this will not address some of the issues).